### PR TITLE
#184752262 User/buyer should be able to add items to their cart

### PR DESCRIPTION
**Pull request by madebypatrick**
#### What does this PR do?
It allows the users to add the products they wish to check out to their carts.

#### What are the relevant pivotal tracker/Trello stories?
[#184752262 ](https://www.pivotaltracker.com/story/show/184752262)

### CODE REVIEW

**Review by Folio123-c — APPROVED** [review]

**Review by Folio123-c — APPROVED** [review]

**Review by NdatimanaSamuel — APPROVED** [review]

**Review by Virgile1k — APPROVED** [review]
checked

File: tests/user/enableDisableAccount.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+} from '../mocks/user.mock';
+import db from '../../src/database/models';
+
+const User = db.users;
+
+let uuid = User.id;
+// const api="http://localhost:3030/api/v1";
+
+
+ //update account status
```
Comment (nshutijonathan) [inline tests/user/enableDisableAccount.test.js]: remove commented out codes

File: tests/user/enableDisableAccount.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+//     });
+
+//    });
+
+ 
+
+// describe(' account Status', () => {
+//   test('enable disable account status', async () => {
+//     const response = await request(app).put(`/api/v1/user/updateAccountStatus/${userUuid}`);
+//     expect(response.status).toBe(200);
```
Comment (nshutijonathan) [inline tests/user/enableDisableAccount.test.js]: remove commented out codes

**Review by nshutijonathan — COMMENTED** [review]
check some comments I left

**Review by Cynthia-kam — APPROVED** [review]

**Review by shemajolivetgislain — APPROVED** [review]

**Review by EdwinBayingana — APPROVED** [review]

**Review by Rod-code — APPROVED** [review]

**Pull request merged**